rest/admin_api.go

```diff
@@ -131,6 +136,19 @@ func (h *handler) handleGetDbConfig() error {
 	includeJavascript, _ := h.getOptBoolQuery("include_javascript", true)
 	_ = includeJavascript
 
+	if h.server.bootstrapContext.connection != nil {
+		found, dbConfig, err := h.server.fetchDatabase(h.db.Name)
+		if err != nil {
+			return err
+		}
+
+		h.response.Header().Set("ETag", dbConfig.Version)
+
+		if !found {
+			return base.HTTPErrorf(http.StatusNotFound, "database config not found")
+		}
```

Comment:
This probably needs reordering to avoid a panic if no `dbConfig` is found?
```suggestion
		if !found {
			return base.HTTPErrorf(http.StatusNotFound, "database config not found")
		}
		h.response.Header().Set("ETag", dbConfig.Version)
```